blog/_posts/2016-08-15-Introducing-Team-Fedex++.md

```diff
@@ -0,0 +1,88 @@
+We are Python lovers. We get a problem and the first way that comes in our mind to solve it is through Python. Choosing PyDSA over any other project was comfortable for us and at the same time challenging considering our ardent love for Algorithms and Python. We have got to give it our best :p
+
+![Team PyDSA](/img/blog/2016/TeamFedex++2.jpeg)
+*All images have been taken from Rails Girls Team App with permissions*   
+
+##### **First Pydelhi Conference**
+* We attended our first ever conference in March when we found out that one of biggest PyDelhi conferences was happening very close to our institute. 
+* Amongst most amazing talks on latest tools and technologies, we realised that participation of women is drastically low in these kind of events.
+* We were so disappointed by the fact that there is such low awareness amongst women and we realised the importance of initiatives like RGSoC to promote girls into this environment.
+* It also happened to be the place where we first met our Mentor, [_**Amit Kumar**_](http://iamit.in/ "Amit Kumar") who also introduced us to two of our coaches, [_**Anuvrat Parashar**_](http://anuvrat.in/ "Anuvrat Parashar") and [_**Saurabh Kumar**_](https://saurabh-kumar.com "Saurabh Kumar").
+
+##### **Coaches and Mentors**
+* We got introduced to our mentor in the PyDelhi Conference who did an amazing job at encouraging us to apply for RGSoC. He even helped us find out coaches. 
+* We found another coach amongst our friends, [_**Ankit Sultana**_](https://ankitsultana.me "Ankit Sultana") and the other one being our professor at IIITD, [_**Dr. Rahul Purandare**_](https://www.iiitd.edu.in/~purandare/ "Dr. Rahul Purandare"). 
+* All our Coaches form a highly enthusiastic group of programmers who never fail to encourage us, are available 24/7 for help and are more than willing to motivate us to continue our journey in Open Source that we have started with RGSoC.
+
+
+![Team-Fedex++ at Work](/img/blog/2016/TeamFedex++3.jpeg)
+*image taken by Sahar Siddiqui* 
+
+
+### **What we have achieved so far?**
+* We designed the basic draft of PyDSA visualisation website. Link to wireframes : <https://goo.gl/8paaNx>
+* We learnt about Pep8 format, Python Tests, Travis CI and a few issues and their solutions.
+* We set up a Django server and made a Test App in it. 
+* As guided by our coaches, we started learning more about d3.js library and tried to code a few visualisations in it. 
+* We learnt how does the interaction between the frontend and backend works using d3.js in frontend and Django framework.
+* We learnt about classes and objects in Python and Design Patterns that should be implemented in order to structure the PyDSA library.
+* Completed the list of algorithms mentioned in the list of algorithms required for the first official release.   
+
+In the process, we have reviewed some Pull Requests made by others and we have sent a few of them ourselves.
```

Comment:
you need two or more spaces at the end of each line of this block so that your list shows up as a list in markdown.